frontend/test/pytest/test_conditionals.py

```diff
@@ -72,6 +67,59 @@ def else_fn():
         assert circuit(5) == 25
         assert circuit(6) == 36
 
+    def test_cond_one_else_if(self):
+        """Test a cond with one else_if branch"""
+
+        @qjit()
```

Comment:
Very minor, and I know some tests do specify the parenthesis, but they aren't needed unless arguments are supplied to `qjit`, so this is cleaner:
```suggestion
        @qjit
```